From 6abd71474677f223192ef60b94bb68a319735044 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 28 Dec 2020 13:26:41 +0700
Subject: [PATCH 2/2] drm: bridge: adv7511: allow support interlaced mode

This commit is created to allow adv7511 support interlaced mode.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 drivers/gpu/drm/bridge/adv7511/adv7511_drv.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c b/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c
index a0d392c338da..2f76661ae4fd 100644
--- a/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c
+++ b/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c
@@ -846,6 +846,8 @@ static int adv7511_connector_init(struct adv7511 *adv)
 		return -ENODEV;
 	}
 
+	adv->connector.interlace_allowed = 1;
+
 	if (adv->i2c_main->irq)
 		adv->connector.polled = DRM_CONNECTOR_POLL_HPD;
 	else
-- 
2.17.1

